src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -658,6 +673,21 @@ public static void Compare(string strA, int indexA, string strB, int indexB, int
             }
             // Use Compare(string, int, string, int, int, StringComparison)
             Assert.Equal(expected, Math.Sign(string.Compare(strA, indexA, strB, indexB, length, comparisonType)));
+
```

Comment:
Can the if (!skipNonComparisonOverloads) part above (L653, L665) be enabled now?

Reply:
Looks like it should be (but we should do it outside this PR). 
cc @tarekgh to confirm.

Reply:
can move part of this new block into the the (!skipNonComparisonOverloads) block but I don't think we can move the whole new block there because !skipNonComparisonOverloads work be when having non null strings and not 0 indexes.

---
In reply to: [177237091](https://github.com/dotnet/corefx/pull/28347#discussion_r177237091) [](ancestors = 177237091)

Reply:
I think Josh was talking about the commented out test:
```C#
// Uncomment when this is exposed in .NET Core (dotnet/corefx#10066)
// Assert.Equal(expected, Math.Sign(string.Compare(strA, indexA, strB, indexB, length, ignoreCase: false)));
```

Reply:
if this is the case, then yes. this can be enabled now.